README.md

```diff
@@ -13,8 +12,24 @@ End-user apps in WSO2 Identity Server
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-is/blob/master/LICENSE)
 [![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
 
+---
+
 ## Prerequisite
 
+### Setup Development Environment
+
+1. Install NodeJS LTS(Latest Stable Version) from [https://nodejs.org/en/download/](https://nodejs.org/en/download/).
+2. Instal the latest version of [pnpm](https://pnpm.io/).
+
+    ```shell
+    npm install -g pnpm@latest
+    ```
+
+    Or, follow the other [recommended installation options](https://pnpm.io/installation).
+
+3. Install Maven from [https://maven.apache.org/download.cgi](https://maven.apache.org/download.cgi).
+4. Install JDK 1.8 [https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html).
+
 ### Download WSO2 Identity Server
```

Comment:
Wouldn't it be better to move the notice about WSO2 IS under prerequisites section?